### DISCUSSION TITLE
Broken link to license here from front page of hypertrace.org

### DISCUSSION THREAD

**Discussion by carehart** [hypertrace/hypertrace#178] [General] [answered]
Folks, [the main site](https://www.hypertrace.org/) has a link in its footer to "license" which fails. It goes to:
https://github.com/hypertrace/hypertrace/blob/master/LICENSE.txt

when it should instead go instead to:

https://github.com/hypertrace/hypertrace/blob/main/LICENSE

Obviously, the top-level branch name here was renamed and somehow that change wasn't propagated to this footer link. 

Apologies if this is not the right place to bring this up, but I couldn't find any way on the main site to report such an issue. (And I looked but could not find that page as something against which I could issue a PR here.) 

Hope that's helpful.

**Comment by dnielsen** [comment, marked as answer]
Thanks @carehart. You hit the nail on the head about the cause. I've updated the license link. Yes, you can submit here. You may also submit via our [Hypertrace Slack Channel](https://hypertrace.slack.com) which you can [join here](https://www.hypertrace.org/get-started).

**Comment by carehart** [comment]
Great, thanks.
